citesphere/src/main/webapp/WEB-INF/views/auth/tokens.jsp

```diff
@@ -0,0 +1,96 @@
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<script src="<c:url value="/resources/notify/bootstrap-notify.min.js" />"></script>
+<link href="<c:url value="/resources/notify/animate.css" />" rel="stylesheet">
+<script>
+$(function() {
+	$(".revoke-access").click(function(event) {
+		var id = $(this).data('client-id');
+		$("#revoke-access-button").attr("data-client-id", id);
+		$("#confirm-client-entry-name").text($(this).data('client-name'));
+		$('#revoke-access-confirmation').modal("show");	
+		event.preventDefault();
+	});
+	$("#revoke-access-button").click(function() {
+		var id = $(this).data("client-id");
+		$.ajax({
+			'url': '<c:url value="/auth/tokens/" />' + id + "?${_csrf.parameterName}=${_csrf.token}",
+			'type': "DELETE",
+			'success': function(data) {
+				$("#tr-" + id).remove();
+				$.notify('<i class="fas fa-check-circle"></i> Access Revoked successfully!', {
+					type: 'success',
+					offset: {
+						x: 50,
+						y: 90
+					},
+					animate: {
+						enter: 'animated fadeInRight',
+						exit: 'animated fadeOutRight'
+					}
+				});
+			},
+			'error': function(data) {
+				$.notify('<i class="fas fa-exclamation-circle"></i> Error Occurred! Access could not be revoked', {
+					type: 'danger',
+					offset: {
+						x: 50,
+						y: 90
+					},
+					animate: {
+						enter: 'animated fadeInRight',
+						exit: 'animated fadeOutRight'
+					}
+				});
+			}
+		});
+		$('#revoke-access-confirmation').modal("hide");
+		event.preventDefault();
+		$(this).removeData();
+	});
+});
+
+</script>
+
+<c:if test="${not empty clientList}">
+	<h2>Authorized OAuth Apps</h2>
+	<br/>
+	<table class="table table-striped" id="table">
+	<tr>
+		<th>App Name</th>
+		<th>Client Id</th>
+		<th/>
+	</tr>
+	<c:forEach items="${clientList}" var="client">
+	<tr id="tr-${client.clientId}"> 
+		<td data-id="${client.name}">${client.name}</td>
+		<td data-id="${client.clientId}">${client.clientId}</td>
```

Comment:
actually, let's show the description not the client id. The id doesn't mean anything to the user.